app.js

```diff
@@ -76,8 +103,51 @@ app.get("*", function (req, res) {
   res.render("notFound");
 });
 
+//Register route
+app.get('/register', (req, res) => {
+  
+});
+
+app.post('/register', async (req, res) => {
+
+  try {
+    const {
+      password,
+      email,
+      name,
```

Comment:
Name and organization are optional, so you might not get them in req.body.user ... if it works, and you tested it, that's fine. If not please add code to handle it.